qds_sdk/cluster.py

```diff
@@ -286,7 +330,9 @@ def _parse_create_update(cls, args, action):
                                  default=None,
                                  help="don't encrypt the ephemeral drives on" +
                                       " the instance",)
-        security_group.add_argument("--customer-ssh-key",
+
+        customer_ssh_key_parameter = "--ssh-public-key" if (api_version >= 1.3) else "--customer-ssh-key"
```

Comment:
Yeah, I know that the API expects the parameter to be named `ssh_public_key` now. But I think the command line flag should still be called `--customer-ssh-key`.

The reason for this is backwards compatibility. At some point in the future, we would change the default version from `v1.2` to `v1.3`.

If we change this command-line flag, all the command lines of the following form would need to be updated:
```
qds.py cluster create --label test_label
                      --access-key-id aki
                      --secret-access-key sak
                      --customer-ssh-key file
```
However, if we keep calling the command-line flag `--customer-ssh-key`, we would maintain command-line backwards compatibility.

I would also suggest that you go through this PR and look for other cases which may have introduced incompatibilities like this and fix them.